styles/utlis.css

```diff
@@ -35,6 +44,41 @@
   top: 0rem;
 }
 
+
+.m3 {
+  margin: 3rem;
+}
+
+.m1 {
```

Comment:
Should these not come from component library?

Reply:
Yes, at the time of developing this, component library code was not merged. will remove this from here.